retrieve-data.js

```diff
@@ -130,12 +132,20 @@ function retrieveContactLanguage(client, msisdn) {
           "Accept": "application/vnd.v1+json"
         },
       }).then(res => res.data.fields.language);
-    }
+}
+
+const rssFeedUrl = "https://www.who.int/rss-feeds/news-english.xml";
+
+async function retrieveLatestNews() {
+  debug("retrieving from RSS");
+  return feed = await parser.parseURL(rssFeedUrl)
```

Comment:
It doesn't seem like this parser does any unescaping of the HTML, and I see that there is some HTML escaping in some of the contentSnippets, eg. `official launch of the initiative.&nbsp;&ldquo;The COVID-19 Technology`

Reply:
Thanks for the reminder!

Reply:
I've added this in my latest commits